client/my-sites/plugins/plugin-item/style.scss

```diff
@@ -7,8 +7,30 @@
 	.plugins-list &.is-compact:last-child {
 		margin-bottom: 0;
 	}
-}
 
+	.notice.is-compact {
+		margin: 0;
+
+		.notice__text {
+			padding: 6px 8px;
+			text-transform: capitalize;
+		}
+	}
+
+	.notice.is-in-progress,
```

Comment:
I'd agree with that. The standard 'Last updated' text doesn't need a status, but when we are updating or deactivating we could make it more visible.